core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -75,6 +75,11 @@
           <div class="protractor-test-conversation-input"
                angular-html-bind="activeCard.interactionHtml">
           </div>
+          <div class="conversation-skin-tutor-card-tooltip">
+              <button class="conversation-skin-tutor-card-tooltip-button">?
+                <span class="tooltip-text">Please don't enter any personal information here. Your answer will be saved and can be read by the creator.</span>
```

Comment:
Use I18N label.

Reply:
Should I add `i18n` as an attribute or label here? I don't know exactly how to do so. Please help! @seanlip

Reply:
@vibhor98, you can take a look [here](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/creator_dashboard/creator_dashboard.html#L19). Also, add this label to assets/i18n/qqq.json and en.json file as such required.

Reply:
@DubeySandeep , I've added the label but it is not rendered. Please help!
![screen shot 2018-02-05 at 10 02 30 pm](https://user-images.githubusercontent.com/26625980/35816437-b92dab04-0ac0-11e8-9d52-a19a49fb3aa3.png)
![screen shot 2018-02-05 at 10 03 01 pm](https://user-images.githubusercontent.com/26625980/35816462-caf84cea-0ac0-11e8-986c-ea839656dab8.png)
I have added it to both `qqq.json` and `en.json`.